packages/bpk-component-split-input/src/BpkSplitInput.module.scss

```diff
@@ -0,0 +1,36 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ /* stylelint-disable */
+
+@import '~bpk-mixins/index';
+
+.BpkSplitInput {
+  display: flex;
+  width: 100%;
+  padding-top: $bpk-spacing-xs;
+  flex-direction: row;
+  align-items: center;
+  font-size: $bpk-font-size-base;
+  direction: ltr;
```

Comment:
will this component work in RTL too?

Reply:
Yes it works in RTL as well and it can be tested in storybook, with RTL on/off. 
<img width="1445" alt="Screenshot 2022-06-21 at 11 52 26" src="https://user-images.githubusercontent.com/107639278/174783036-a7934580-a360-4c38-aa80-29ca097429ef.png">

Reply:
in RTL, are you supposed to start from the right-most cell and go left, or would it work the same as LTR?
as the focus does not move to the next cell after you type a digit when starting from the right-most cell